packages/client/src/index.ts

```diff
@@ -139,4 +143,36 @@ export class TalentLayerClient {
       this.chainId,
     );
   }
+
+  /**
+   * Provides access to user functionalities.
+   * @type {IUser}
+   */
+
+  // @ts-ignore
+  get user(): IUser {
```

Comment:
user is called profile here to make it more "user friendly", so this already exist above